### PR TITLE
Updates configs from CircleCI

### PR DESCRIPTION
**Pull request by pauloedurezende**
**CircleCI:** Updates the configs from CircleCI to use your API version 2.0
**Coveralls:** Adds Coveralls config file to specify your repo token in to be able to send test info

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8.11.2
+    steps:
+      - checkout
+      - run: npm install
+      - run: npm test
+      - run: npm run coveralls
```
Comment (robertoachar) [inline .circleci/config.yml]: Please, remove this command. Travis will do the job.

File: .coveralls.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+repo_token:
```
Comment (robertoachar) [inline .coveralls.yml]: I guess we don't need this file. Travis knows how to accomplish the job.

**Review by robertoachar — CHANGES_REQUESTED** [review]
Hey, thanks for the contribution. Could you please consider these changes?

**Comment by robertoachar** [comment]
@pauloedurezende We need same changes on templates too. I can help you with that.

File: .circleci/config.yml (hunk 1 above)
Comment (pauloedurezende) [reply .circleci/config.yml]: Following the CircleCI documentation version 2.0, this codes is necessary to make the builds. If you remove all this commands, CircleCI return a following error: ```jobs: build: required key [steps] not found```, so, on this part of the configuration, is needed at least one command.

So, if you go replace Travis to CircleCI, this commands ends up being necessary to lint, test and send the coveralls test results to Coveralls.io

File: .coveralls.yml (hunk 2 above)
Comment (pauloedurezende) [reply .coveralls.yml]: When you work with **Travis + Coveralls**, all informations about the coverage test are sent with no problems to Coveralls.io.

But, if you work with **CircleCI + Coveralls**, when we execute ```npm run coveralls```, the Coveralls package return the following error: ```Bad response: 422 {"message": "Couldn't find a repository matching this job.", "error": true}```, then, referring to the documentation (that also a bit confused), Coveralls recommends to create the ```.coveralls.yml``` file to refer the repository token and be able to send the coveralls test result to Coveralls.io

**Comment by pauloedurezende** [comment]
@robertoachar, Right! I appreciate if you help me with the changes on templates!

File: .circleci/config.yml (hunk 1 above)
Comment (robertoachar) [reply .circleci/config.yml]: You're right, but I asked you to remove only this command: `- run: npm run coveralls`. Travis already sends the results to Coveralls and it doesn't need the token. :wink:

Travis configuration file:

```yaml
after_script:
  - npm run coveralls
```

You got it?

File: .coveralls.yml (hunk 2 above)
Comment (robertoachar) [reply .coveralls.yml]: Indeed. Same as before, let Travis do the job.

**Comment by robertoachar** [comment]
@pauloedurezende Sure, I'll help you. Let me know if you understood my requests, so we can move on to templates. Thanks again, mate!

**Comment by pauloedurezende** [comment]
@robertoachar  Okay, now I understand everything, I've made the changes you've requested!

**Review by robertoachar — APPROVED** [review]

**Pull request merged**